### DISCUSSION TITLE
Minor bug in CTkEntry class code

### DISCUSSION THREAD

**Discussion by jh-jensen** [TomSchimansky/CustomTkinter#1300] [General]
Hello. I noticed the placeholder text for entry widgets disappears whenever you add a textvariable attribute. 
I fixed it by changing line 74 and 300:

`self._textvariable == ""`

to 

`self._textvariable.get() == ""`

since self._textvariable returns PYVAR(number) and not the value of the textvariable itself, which I assume is what was originally intended (it's written correctly on line 119).

**Comment by TomSchimansky** [comment]
This is intentional, in some cases there were problems when you keep the placeholder with the textvariable.

**Reply by jh-jensen** [reply]
I see, in which specific cases might problems arise? Would be nice to know beforehand since I'd like to keep the placeholder text
